src/test/java/core/basesyntax/ActivityServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import static org.hamcrest.CoreMatchers.startsWith;
+
+import core.basesyntax.strategy.ActivityService;
+import core.basesyntax.strategy.ActivityStrategy;
+import core.basesyntax.strategy.impl.BalanceService;
+import core.basesyntax.strategy.impl.PurchaseService;
+import core.basesyntax.strategy.impl.ReturnService;
+import core.basesyntax.strategy.impl.SupplyService;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ActivityServiceTest {
+
+    private static final Map<String, ActivityService> activityHandlerMap = new HashMap<>();
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testActivityService_wrongActivity() {
+
+        activityHandlerMap.put("b", new BalanceService());
```

Comment:
```suggestion
    public void testActivityService_wrongActivity() {
        activityHandlerMap.put("b", new BalanceService());
```
we don't need an empty line between method signature and its body